Makefile

```diff
@@ -39,3 +39,13 @@ bench-obs-logs-stage: $(GINKGO) $(PROMETHEUS) $(EMBEDMD) $(REPORT_DIR)
 	OBS_LOKI_ING="observatorium-loki-ingester" \
 	./run.sh
 .PHONY: bench-staging
+
+bench-obs-logs-test: $(GINKGO) $(PROMETHEUS) $(EMBEDMD) $(REPORT_DIR)
+	@TARGET_ENV=ocp-observatorium-test \
+	OBS_NS=observatorium-logs-test \
+	OBS_LOKI_QF="observatorium-loki-query-frontend" \
+	OBS_LOKI_QR="observatorium-loki-querier" \
+	OBS_LOKI_DST="observatorium-loki-distributor" \
+	OBS_LOKI_ING="observatorium-loki-ingester" \
+	./run.sh
+.PHONY: bench-ocp-test
```

Comment:
The phony directive requires to be named after its actual target, `bench-ocp-test` -> `bench-obs-logs-test`.

Reply:
fixed